### PR TITLE
Add background to js visualisation.

### PR DESCRIPTION
**Pull request by MichaelMauderer**
### Pull Request Description

Avoids overlapping JS visualizations adding a background.
This fixes some occlusion issues.

### Important Notes
This in only a partial solution to #867, proper solution requires #913 to be finalized and implemented.

### Checklist
Please include the following checklist in your PR:

- [x] The documentation has been updated if necessary.
- [x] All code conforms to the [Rust](https://github.com/luna/enso/blob/main/docs/style-guide/rust.md) style guide.
- [x] All code has been tested where possible.
- [x] All code has been profiled where possible.

### CODE REVIEW

File: src/rust/ide/view/graph-editor/src/component/visualization/foreign/java_script/instance.rs [hunk 1]
```diff
@@ -95,11 +97,22 @@ pub struct InstanceModel {
 
 impl InstanceModel {
 
-    fn create_root() -> result::Result<DomSymbol, Error> {
+    fn get_background_color(scene:&Scene) -> color::Rgba {
+        let styles     = StyleWatch::new(&scene.style_sheet);
+        let bg_color   = styles.get_color(ensogl_theme::vars::graph_editor::visualization::background::color);
```
Comment (wdanilo) [inline src/rust/ide/view/graph-editor/src/component/visualization/foreign/java_script/instance.rs:102]: alignment

**Review by wdanilo — APPROVED** [review]

File: src/rust/ide/view/graph-editor/src/builtin/visualization/native/raw_text.rs [hunk 2]
```diff
@@ -89,10 +89,14 @@ impl RawTextModel {
         let _red       = text_color.red * 255.0;
         let _green     = text_color.green * 255.0;
         let _blue      = text_color.blue * 255.0;
+
+        let bg_color   = styles.get_color(ensogl_theme::vars::graph_editor::visualization::background::color);
+        let bg_color   = color::Rgba::from(bg_color);
         let text_color = format!("rgba({},{},{},{})",_red,_green,_blue,text_color.alpha);
+        let bg_hex     = format!("rgba({},{},{},{})",bg_color.red*255.0,bg_color.green*255.0,bg_color.blue*255.0,bg_color.alpha);
```
Comment (farmaazon) [inline src/rust/ide/view/graph-editor/src/builtin/visualization/native/raw_text.rs:96]: long line

**Review by farmaazon — COMMENTED** [review]
I see one little "regression": Before the shadow of visualizations was visible over another:
image

Now It disappears:

image

@wdanilo I'm not sure how it is important, but looks rather odd.

**Comment by wdanilo** [comment]
Regarding lack of shadows - I was talking it will happen few days ago. I believe that non overlapping viz is more important than shadow error, so Im ok with keeping it this way :)

**Review by farmaazon — APPROVED** [review]
So let it be.

**Comment by MichaelMauderer** [comment]
I actually added a fix for the shadow. Please have another quick look @farmaazon @wdanilo

**Review by farmaazon — APPROVED** [review]
Checked - it works! Nice done!

**Review by farmaazon — APPROVED** [review]

File: src/rust/ide/view/graph-editor/src/component/visualization/container.rs [hunk 3]
```diff
@@ -41,7 +41,7 @@ use ensogl_theme as theme;
 
 const DEFAULT_SIZE      : (f32,f32) = (200.0,200.0);
 const CORNER_RADIUS     : f32       = super::super::node::CORNER_RADIUS;
-const SHADOW_SIZE       : f32       = super::super::node::SHADOW_SIZE;
+const SHADOW_SIZE       : f32       = 0.0 * super::super::node::SHADOW_SIZE;
```
[the remark this answers is not in the document]
Comment (MichaelMauderer) [reply src/rust/ide/view/graph-editor/src/component/visualization/container.rs:44]: Yes. I'll add a comment there. I wanted to keep it like this, as this just "deactivated" the shadow here, while we use the CSS replacement. But the correct reference value here should be `super::super::node::SHADOW_SIZE` and that is more obvious like this. (And with a comment)

**Pull request merged**